### PR TITLE
small improvements to the encrypted secret doc

### PR DESCRIPTION
**Pull request by arbll**
### What does this PR do?

A few improvements that I think would be nice to have.

### CODE REVIEW

File: docs/agent/secrets.md
Comment (olivielpeau) [inline docs/agent/secrets.md]: `to pick up new configuration changes` -> `to pick up changes on config files` (just want to mention that it needs to be restarted for files but not for other config sources)

**Review by olivielpeau — DISMISSED** [review]
🚀 

just a nit

let's wait for a review from @hush-hush too :)

File: docs/agent/secrets.md
Comment (ngetahun-dd) [inline docs/agent/secrets.md]: ~passowrd~ password

File: docs/agent/secrets.md
Comment (ngetahun-dd) [inline docs/agent/secrets.md]: ~passowrd~ password

**Review by hush-hush — APPROVED** [review]

**Pull request merged**
